test/functional/p2p_filter.py

```diff
@@ -51,7 +61,6 @@ def on_merkleblock(self, message):
     def on_tx(self, message):
         self.tx_received = True
 
-
```

Comment:
style-nit (feel free to ignore): I think pep-8 says two new lines before `class`